lib/puppet/type/posix_acl.rb

```diff
@@ -118,6 +118,9 @@ def self.descendant?(a, b)
   autorequire(:package) do
     ['acl']
   end
+  autorequire(:file) do
+    self[:path]
+  end
```

Comment:
Is it possible to call `autorequire(:file)` twice? Perhaps line 107 should be modified if `autorequire_type` is `file` it shouldn't use `path.parent` but rather `path`?